storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -108,7 +108,15 @@
      */
     public static final String STORM_MESSAGING_NETTY_AUTHENTICATION = "storm.messaging.netty.authentication";
     public static final Object STORM_MESSAGING_NETTY_AUTHENTICATION_SCHEMA = Boolean.class;
-
+    
+    /**
+     * Netty based messaging: A quality of protection (qop) value is one of "auth" - authentication only, 
+     * "auth-int" - authentication plus integrity protection, "auth-conf" - authentication plus integrity and confidentiality protection.
+     * The default qop is "auth".
+     */
+    public static final String STORM_MESSAGING_NETTY_PROTECTION = "storm.messaging.netty.protection";
+    public static final Object STORM_MESSAGING_NETTY_PROTECTION_SCHEMA = String.class;
```

Comment:
* Is this config intended to be a cluster-wide setting or could it be different for each topology?
* It would be nice to validate the setting is one of the accepted values beyond being a string.